pkg/debug/scripts/collect-info.sh

```diff
@@ -155,6 +155,36 @@ collect_network_info()
         dhcpcd -U -4 "$iface"
     done > "$DIR/network/dhcpcd-all-ifaces" 2>&1
 
+    echo "   - cellular modems"
+    MODEMS="$(eve exec --fork wwan mmcli -L |\
+              sed -n 's/.*\/ModemManager1\/Modem\/\([0-9]\+\).*/\1/p' | uniq)"
+    for MODEM in $MODEMS; do
+        INFO="$(eve exec --fork wwan mmcli -m "$MODEM")"
+        echo
+        echo "Modem $MODEM:"
+        echo "$INFO"
+        SIMS="$(echo "$INFO" |\
+                sed -n 's/.*\/ModemManager1\/SIM\/\([0-9]\+\).*/\1/p' | uniq)"
+        for SIM in $SIMS; do
```

Comment:
Doesn't make sense to put a message here if no SIM card is inserted?

Reply:
No need, this information will be obvious from `mmcli -m "$MODEM"` (aka `echo $INFO` in the script)

Reply:
Notice "failed reason: sim-missing"